book/caching.md

```diff
@@ -570,22 +553,20 @@ parameters:
 
 NOTE: If the parameters block is already present in the yml file, append the twig.config block to it.
 
-1. Afterwards rebuild the Drupal cache with `drush cr` otherwise your website will encounter an unexpected error on page reload.
-
+Afterwards rebuild the Drupal cache with `drush cr` otherwise your website will encounter an unexpected error on page reload.
 
 ## Reference
 
 * [Drupal: cache tags for all, regardles of your backend From Matt Glaman 22, August 2022](https://mglaman.dev/blog/drupal-cache-tags-all-regardless-your-backend)
 * [Cache contexts overview on drupal.org](https://www.drupal.org/docs/drupal-apis/cache-api/cache-contexts)
 * [Caching in Drupal 8 a quick overview of Cache tags, cache context and cache max-age with simple examples](https://zu.com/articles/caching-drupal-8)
 * [Nedcamp video on caching by Kelly Lucas from November 2018](https://www.youtube.com/watch?v=QCZe2K13bd0&list=PLgfWMnl57dv5KmHaK4AngrQAryjO_ylaM&t=0s&index=16)
-
+* [#! code: Drupal 9: Debugging Cache Problems With The Cache Review Module](https://www.hashbangcode.com/article/drupal-9-debugging-cache-problems-cache-review-module)
```

Comment:
Thanks for these.  I've started posting the dates for the linked articles.  I think it is helpful to see if something is pretty recent vs ancient.  What do you think?

Reply:
Good idea, I didn't notice that in the links but I can add them in 👍

Reply:
I came up with that recently, so it probably isn't that way everywhere.  As I go I am formulating ideas.  I probably should add a style guide or something outlining these ideas.  Didn't you and I talk about that?

Reply:
We did, I had just listened to a talk about technical writing and style guides. I'll dig out the slides as it was a really good talk.